onnx/test/shape_inference_test.py

```diff
@@ -3862,7 +3876,7 @@ def test_optional_tensor_has_element(self) -> None:
              make_node('OptionalHasElement', ['sequence'], ['output'])],
             [])
         self._assert_inferred(graph, [optional_val_info,
-                                      make_tensor_value_info('output', TensorProto.BOOL, None)])  # type: ignore
+                                      make_tensor_value_info('output', TensorProto.BOOL, ())])  # type: ignore
```

Comment:
Yes, here this looks fine, since `output` must be a scalar boolean with empty shape